clients/camunda-spring-boot-starter/src/main/java/io/camunda/client/spring/actuator/JobWorkerController.java

```diff
@@ -0,0 +1,226 @@
+import io.camunda.client.jobhandling.JobWorkerChangeSet.RequestTimeoutChangeSet;
+import io.camunda.client.jobhandling.JobWorkerChangeSet.StreamEnabledChangeSet;
+import io.camunda.client.jobhandling.JobWorkerChangeSet.StreamTimeoutChangeSet;
+import io.camunda.client.jobhandling.JobWorkerChangeSet.TenantIdsChangeSet;
+import io.camunda.client.jobhandling.JobWorkerChangeSet.TimeoutChangeSet;
+import io.camunda.client.jobhandling.JobWorkerChangeSet.TypeChangeSet;
+import io.camunda.client.jobhandling.JobWorkerManager;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.springframework.boot.actuate.endpoint.annotation.ReadOperation;
+import org.springframework.boot.actuate.endpoint.annotation.Selector;
+import org.springframework.boot.actuate.endpoint.annotation.WriteOperation;
+import org.springframework.boot.actuate.endpoint.web.annotation.WebEndpoint;
+import org.springframework.lang.Nullable;
+
+@WebEndpoint(id = "jobWorkers")
+public class JobWorkerController {
+  private final JobWorkerManager jobWorkerManager;
+
+  public JobWorkerController(final JobWorkerManager jobWorkerManager) {
+    this.jobWorkerManager = jobWorkerManager;
+  }
+
+  @ReadOperation
+  public List<JobWorkerDto> getJobWorkers() {
+    return jobWorkerManager.getJobWorkers().entrySet().stream()
+        .map(e -> fromJobWorkerValue(e.getKey(), e.getValue()))
+        .toList();
+  }
+
+  @ReadOperation
+  public JobWorkerDto getJobWorker(@Selector final String type) {
+    return fromJobWorkerValue(type, jobWorkerManager.getJobWorker(type));
+  }
+
+  @WriteOperation
+  public void updateJobWorkers(
+      @Nullable final String name,
+      @Nullable final Boolean enabled,
+      @Nullable final String tenantIds,
+      @Nullable final Duration timeout,
+      @Nullable final Integer maxJobsActive,
+      @Nullable final Duration requestTimeout,
+      @Nullable final Duration pollInterval,
+      @Nullable final String fetchVariables,
+      @Nullable final Boolean forceFetchAllVariables,
+      @Nullable final Boolean streamEnabled,
+      @Nullable final Duration streamTimeout,
+      @Nullable final Integer maxRetries,
+      @Nullable final Boolean reset,
+      @Nullable final Boolean applyCustomizers) {
+    if (reset != null && reset) {
+      jobWorkerManager.resetJobWorkers();
+    } else {
+      jobWorkerManager.updateJobWorkers(
+          createChangeSet(
+              null,
+              name,
+              enabled,
+              tenantIds,
+              timeout,
+              maxJobsActive,
+              requestTimeout,
+              pollInterval,
+              fetchVariables,
+              forceFetchAllVariables,
+              streamEnabled,
+              streamTimeout,
+              maxRetries),
+          ofNullable(applyCustomizers).orElse(false));
+    }
+  }
+
+  @WriteOperation
+  public void updateJobWorker(
+      @Selector final String finalType,
```

Comment:
🔧minor, name suggestion:  `typeSelector`

Reply:
Good idea :) I was a bit unsure about the naming actually.